src/server.js

```diff
@@ -14,13 +13,13 @@ mongoose
   .then(() => console.log('Connected to MongoDB successfully'))
   .catch((err) => console.log('Could not connect to MongoDB', err));
 
-//Create a basic route
-app.get('/', (req, res) => {
-  res.send('Hello World!');
-});
-
+app.use(express.urlencoded());
 app.use(express.json());
 
+// Use route in folder routes
+const userRoute = require('./routes/userRoute');
```

Comment:
Import routes in top of file under mongoose for example

Reply:
i'll change it